src/libServer/ScillaIPCServer.cpp

```diff
@@ -15,64 +15,78 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#include "ScillaIPCServer.h"
+#include <jsonrpccpp/server/connectors/unixdomainsocketserver.h>
+
 #include "libPersistence/ContractStorage2.h"
 #include "libUtils/DataConversion.h"
 
-using namespace jsonrpc;
```

Comment:
I understand the benefits of not polluting the global namespace, but given the number of times we have to use `jsonrpc::__` in the code I'm not sure it was worth removing this?

Reply:
I don't have a strong opinion either way on this. I'll update to add the `using namespace jsonrpc` back.